### ISSUE TITLE
Feature wish: enable editing of Time Profile Analysis of a population simulation.

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/PK-Sim#1005]
There is no option to (un-)select the shown data or to change the axis scaling (very annoying!) of the population simulation results chart.

**Comment by msevestre** [comment]
Data as Observed Data? I think there is a tab where this can be removed

And Scaling is defined in the plot configuration (Output selection, Lin log). Right click on your chart=>Edit

**Comment by PavelBal** [comment]
>Data as Observed Data? I think there is a tab where this can be removed

Observed or simulated.

>And Scaling is defined in the plot configuration (Output selection, Lin log). Right click on your chart=>Edit

OK, didnt know about this option. There the data to be shown can also be selected. But it is not so intuitive, as in other charts (individual simulations) the axis can be edited separately.

**Comment by msevestre** [comment]
@PavelBal Fair enough. The Pop Plots are also much more complicated (because of grouping etc.. .so we will never get the flexibility of single curve...or at least not soon) That being said Axis edit such as log/lin etc... should be fairly easy to implement
